cypress/e2e/circulation-log/verify-service-point-in-circ-history.cy.js

```diff
@@ -0,0 +1,112 @@
+import moment from 'moment';
+
+import { DevTeams, Permissions, TestTypes } from '../../support/dictionary';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import UserEdit from '../../support/fragments/users/userEdit';
+import TopMenu from '../../support/fragments/topMenu';
+import Users from '../../support/fragments/users/users';
+import { Locations } from '../../support/fragments/settings/tenant/location-setup';
+import CheckInActions from '../../support/fragments/check-in-actions/checkInActions';
+import InventorySearchAndFilter from '../../support/fragments/inventory/inventorySearchAndFilter';
+import ItemRecordView from '../../support/fragments/inventory/item/itemRecordView';
+import ConfirmItemInModal from '../../support/fragments/check-in-actions/confirmItemInModal';
+
+describe('Circulation log', () => {
+  const testData = {
+    folioInstances: InventoryInstances.generateFolioInstances(),
+    servicePoint: ServicePoints.getDefaultServicePointWithPickUpLocation(),
+    servicePointS1: ServicePoints.getDefaultServicePointWithPickUpLocation(),
```

Comment:
please choose better variable names, it's unclear what S1 means in servicePointS1